snap/snapcraft.yaml

```diff
@@ -69,6 +70,9 @@ parts:
         bin/craftctl: libexec/rockcraft/craftctl
     override-build: |
       ${SNAP}/libexec/snapcraft/craftctl default
+
+      version="$(python3 setup.py --version)"
+      ${SNAP}/libexec/snapcraft/craftctl set version="$version"
```

Comment:
I tested this by (locally) creating a signed tag, building the snap and verifying that the version is correct